code/core/write_sdl_ntuple.cc

```diff
@@ -6,8 +6,544 @@
 #define MAX_NTRIPLET_PER_MODULE 1170
 #define MAX_NQUINTUPLET_PER_MODULE 513
 
+
 //________________________________________________________________________________________________________________________________
 void createOutputBranches()
+{
+    createOutputBranches_v2();
+}
+
+//________________________________________________________________________________________________________________________________
+void fillOutputBranches(SDL::Event* event)
+{
+    fillOutputBranches_v2(event);
+}
```

Comment:
... so idk. probably not...? but i wasn't sure... similar to previous comment.

Reply:
Contrary to my reply in the previous comment, I would be in favor of nuking those. These seem simple enough to reconstruct their pieces, if we need them in the future (which I doubt, since your updates are both fast and complete). And if the removal happens in a single comment, it would be easy to just check that and understand what we were doing.